cmd/server/cadence.go

```diff
@@ -36,7 +37,9 @@ var validServices = []string{historyService, matchingService, frontendService, w
 // main entry point for the cadence server
 func main() {
 	app := buildCLI()
-	app.Run(os.Args)
+	if err := app.Run(os.Args); err != nil {
+		log.Fatal("failed to run entry point for cadence server", tag.Error(err))
```

Comment:
This won't work. Our tagging only works with the special logger. I think you can just :
```
log.Fatal("failed to run entry point for cadence server, err:", err)
```